### ISSUE TITLE
bit Depth is 16bit?

### ISSUE THREAD

**Issue by erfanam7** [dropcreations/Manzana-Apple-Music-Downloader#17]
is the files that we gonna download are 24bit or 16bit depth? i mean, we need the hi-res music. Is it possible to download this quality or is it way much lower than that?
thanks

**Comment by dropcreations** [comment]
@erfanam7 No, this only downloads the 256kbps AAC audios. This is not for ALAC (Hi-Res) audios
